--- symbols_core.cpp	
+++ symbols_core.cpp
@@ -118,6 +118,7 @@
           return nothing::resolve_frames(frames);
          #endif
         #endif
+        return {};
     }
 
     std::vector<stacktrace_frame> resolve_frames(const std::vector<frame_ptr>& frames) {
@@ -149,6 +150,7 @@
           return nothing::resolve_frames(frames);
          #endif
         #endif
+        return {};
     }
 }
 }
